profiling/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "datadog-php-profiling"
-version = "0.15.0"
+version = "0.87.0-alpha.1"
```

Comment:
Is it necessary to keep _master_ on a version number, which needs to regularly updated? I'd strongly prefer having something like `1.0.0-nightly` here on master, if possible.
It's not like we're distributing this via cargo anyway...

Reply:
Customers sometimes run builds from master. I would prefer them to have a real but pre-release version number than being `1.0.0-nightly` all the time.

Reply:
@morrisonlevi Would it be possible to instead insert the version in the circleci checkout step (for tracer as well then, replacing it in the 3 relevant files via `sed`), or are we talking rather about manual builds from source?

Reply:
We discussed in person to change our release process so that the version on master is the latest feature release. We'll append `+$gitsha` e.g. `0.86.0+a885854666d58b3e06792856f8d32f0a2459bafb` in CI when the branch doesn't start with `ddtrace-`. For PECL we'll have to replace `+` with `-` or just strip it. These PECL builds aren't released anyway, so it doesn't exactly matter, but it will choke on `+`.